Project.toml

```
@@ -19,10 +19,11 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 CategoricalArrays = "0.10"
 ColorTypes = "0.10.3, 0.11"
 ComputationalResources = "0.3.2"
-Flux = "0.13, 0.14"
+Flux = "0.14"
 MLJModelInterface = "1.1.1"
-Metalhead = "0.8"
+Metalhead = "0.9"
 ProgressMeter = "1.7.1"
+StatisticalMeasures = "0.1"
 Tables = "1.0"
 julia = "1.6"
```

Since Flux is a dep, and Flux needs julia 1.9 (and we are no longer testing under julia 1.6)

```suggestion
julia = "1.9"
```